doc/versions.json

```diff
@@ -1,6 +1,9 @@
 [
     "2.6.99",
+    "2.6.99-cs",
+    "2.6.1",
```

Comment:
Do we need more files (like e.g. release notes) for NCS 2.6.1 ?

Reply:
yes, just added